### PR TITLE
Remove USE_ETW Compilation Constant from System.Diagnostics.Tracing.Tests

### PR DESCRIPTION
**Pull request by sgaliamov**
Fix for #27116. BasicEventSourceTests.FuzzyTests only.

### CODE REVIEW

**Comment by ViktorHofer** [comment]
Thanks 👍 Do you also want to look at the other files and try to remove USE_ETW as part of this PR?

**Comment by sgaliamov** [comment]
@ViktorHofer first I want to make sure that all builds will pass and choosed direction is right.

**Comment by ViktorHofer** [comment]
works for me. thanks!

File: src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/FuzzyTests.Windows.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
```
[the remark this answers is not in the document]
Comment (sgaliamov) [reply src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/FuzzyTests.Windows.cs]: yep

[the remark this answers is not in the document]
Comment (brianrob) [reply src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/FuzzyTests.Windows.cs]: Do you need to have this test live in a separate file that is only compiled on Windows?  I was hoping that you'd be able to just add the platform detection occur in the ConditionalFactAttribute and that would be enough to excise these tests from unix.

File: src/System.Diagnostics.Tracing/tests/System.Diagnostics.Tracing.Tests.csproj [hunk 2]
```diff
@@ -11,6 +11,15 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Condition="'$(TargetsWindows)' == 'true'" Include="BasicEventSourceTest\FuzzyTests.Windows.cs">
+      <DependentUpon>FuzzyTests.cs</DependentUpon>
```
Comment (brianrob) [inline src/System.Diagnostics.Tracing/tests/System.Diagnostics.Tracing.Tests.csproj]: Do you need the <DependUpon> tags for any of these?

**Comment by brianrob** [comment]
@sgaliamov thanks for contributing this!  I left a few questions for you.

**Comment by brianrob** [comment]
@sgaliamov this looks great.  Can you rebase this PR so that it can be merged?

**Comment by sgaliamov** [comment]
@brianrob there is some things left to do. If you are not hurry, let me finish it.

**Comment by brianrob** [comment]
Sure, let me know when you're ready and I will review this again.

**Comment by sgaliamov** [comment]
@brianrob I'm done.
Some comments to explain my decisions:
1. Just remove **#if** not possible, because some code won't compile on Linux, for example:
    ```
    using Microsoft.Diagnostics.Tracing.Session;
    ```
    That why I've putted ETW-specific code in separate files (_*.Etw.cs_) and marked them with _Condition="'$(TargetGroup)' == 'netcoreapp' And '$(TargetsWindows)' == 'true'"_ in the _csproj_ file.
1. I tried to keep structure of the code to not break tests and logic. That why I used partial method to inject ETW-specific code. I know that partial classes and methods in general are used mostly for auto-generated code, but this case it went well.

**Pull request merged**